contracts/contracts/strategies/BaseCurveAMOStrategy.sol

```diff
@@ -133,14 +141,17 @@ contract BaseCurveAMOStrategy is InitializableAbstractStrategy {
      * well within that abstraction.
      * @param _rewardTokenAddresses Address of CRV
      * @param _assets Addresses of supported assets. eg WETH
+     * @param _maxSlippage Maximum slippage allowed for adding/removing liquidity from the Curve pool.
      */
     function initialize(
         address[] calldata _rewardTokenAddresses, // CRV
-        address[] calldata _assets // WETH
+        address[] calldata _assets, // WETH
+        uint256 _maxSlippage
     ) external onlyGovernor initializer {
         require(_assets.length == 1, "Must have exactly one asset");
         require(_assets[0] == address(weth), "Asset not WETH");
 
+        maxSlippage = _maxSlippage;
```

Comment:
Would prefer making `setMaxSlippage` public and calling through that (or have an internal counterpart that's called in both places). The thing is `setMaxSlippage` has a max value check that's missing here

Reply:
Fixed in this commit: https://github.com/OriginProtocol/origin-dollar/pull/2384/commits/6711b5a8027aeb69f52d50a49bc1a1dde222f4cb.